utils/sqlite_clean.js

```diff
@@ -16,8 +17,12 @@ if (config.sqlite) {
       console.error(e);
       return;
     }
+    let sqlitePath = new Sqlite3(path.join(config.datapath));
+    if (fs.existsSync(path.join(config.datapath, 'sqlite'))) {
+      sqlitePath = new Sqlite3(path.join(config.datapath, 'sqlite'));
+    }
```

Comment:
sqlitePath should be a String instead of Sqlite3 object
 ```diff
-   let sqlitePath = new Sqlite3(path.join(config.datapath));
-    if (fs.existsSync(path.join(config.datapath, 'sqlite'))) {
-      sqlitePath = new Sqlite3(path.join(config.datapath, 'sqlite'));
-    }
+   let sqlitePath = path.join(config.datapath);
+    if (fs.existsSync(path.join(config.datapath, 'sqlite'))) {
+      sqlitePath = path.join(config.datapath, 'sqlite');
+    }
```

Reply:
Done